cql3/restrictions/statement_restrictions.cc

```diff
@@ -839,7 +845,7 @@ bool single_column_restriction::contains::is_satisfied_by(bytes_view collection_
                 }
             } else {
                 auto data_map = value_cast<map_type_impl::native_type>(deserialized);
-                if (!exists_in(data_map | boost::adaptors::transformed([] (auto&& p) { return p.second; }))) {
+                if (!exists_in(data_map | transformed([] (auto&& p) { return p.second; }))) {
                     return false;
```

Comment:
Gratuitous

Reply:
Yeah, the global search-and-replace was too powerful.  But `boost::adaptors::` was also gratuitous, and I'm too lazy to start a one-line PR on master just for this.